cgmes/cgmes-conversion/src/test/resources/cim14/ieee14.xiidm

```diff
@@ -94,7 +94,7 @@
             <iidm:load id="_LOAD___4_EC" name="LOAD   4" loadType="UNDEFINED" p0="47.79999923706055" q0="-3.9000000953674316" bus="_BUS____4_TN" connectableBus="_BUS____4_TN" p="47.79999923706055" q="-3.9000000953674316"/>
         </iidm:voltageLevel>
         <iidm:twoWindingsTransformer id="_BUS____4-BUS____9-1_PT" name="BUS    4-BUS    9-1" r="0.0" x="1.0591881275177002" g="0.0" b="0.0" ratedU1="69.0" ratedU2="13.800000190734863" bus1="_BUS____4_TN" connectableBus1="_BUS____4_TN" voltageLevelId1="_BUS____4_VL" bus2="_BUS____9_TN" connectableBus2="_BUS____9_TN" voltageLevelId2="_BUS____9_VL">
-            <iidm:ratioTapChanger lowTapPosition="1" tapPosition="4" loadTapChangingCapabilities="false">
+            <iidm:ratioTapChanger lowTapPosition="1" tapPosition="4" loadTapChangingCapabilities="false" regulating="false">
```

Comment:
The rule before was nothing means regulating at false ?

Reply:
Yes.